src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -86,10 +86,9 @@ class AddressValidationModal extends React.Component {
       return (
         <button
           className="usa-button-primary"
-          onClick={() => {
-            this.props.closeModal();
```

Comment:
this line could be left, right? and _maybe_ could help with screen reader bugs?

Reply:
@erikphansen We're hoping that actively managing focus will handle the screen reader bugs. The working hypothesis is the focus was being set back to `<body>` so there was nothing for screen readers to interpret as the app state changed.